src/org/labkey/remoteapi/issues/IssueModel.java

```diff
@@ -0,0 +1,217 @@
+    {
+        return setProp(ModelKeys.title, title);
+    }
+
+    // per https://www.labkey.org/Documentation/wiki-page.view?name=sampleJSscripts#issues, for a single notify user
+    // you can supply an email address as a string; for multiple notifies, userIDs are necessary
+    public String getNotify()
+    {
+        return (String) _properties.get(ModelKeys.notifyList.toString());
+    }
+
+    /**
+     * when notify is just 1 user, you can provide their email
+     * @param notify
+     * @return
+     */
+    public IssueModel setNotify(String notify)
+    {
+        return setProp(ModelKeys.notifyList, notify);
+    }
+
+    public Integer getAssignedTo()
+    {
+        return (Integer) getProp(ModelKeys.assignedto);
+    }
+
+    public IssueModel setAssignedTo(Long assignedTo)
+    {
+        return setProp(ModelKeys.assignedto, assignedTo);
+    }
+
+    public String getIssueDefName()
+    {
+        return (String) getProp(ModelKeys.issueDefName);
+    }
+
+    public IssueModel setIssueDefName(String issueDefName)
+    {
+        return setProp(ModelKeys.issueDefName, issueDefName);
+    }
+
+    public String getType()
+    {
+        return (String) getProp(ModelKeys.type);
+    }
+
+    public IssueModel setType(String type)
+    {
+        return setProp(ModelKeys.type, type);
+    }
+
+    public Long getPriority()
+    {
+        return (Long) getProp(ModelKeys.priority);
+    }
+
+    public IssueModel setPriority(Long priority)
+    {
+        return setProp(ModelKeys.priority, priority);
+    }
+
+    public String getComment()
+    {
+        return (String) getProp(ModelKeys.comment);
+    }
+
+    public IssueModel setComment(String comment)
+    {
+        return setProp(ModelKeys.comment, comment);
+    }
+
+    public String getResolution()
+    {
+        return (String) getProp(ModelKeys.resolution);
+    }
+
+    public IssueModel setResolution(String resolution)
+    {
+        return setProp(ModelKeys.resolution, resolution);
+    }
+
+    public IssueModel setProp(ModelKeys propKey, Object value)
+    {
+        return setProp(propKey.toString(), value);
+    }
+
+    public IssueModel setProp(String propName, Object value)
+    {
+        _properties.put(propName, value);
+        return this;
+    }
+
+    public Object getProp(ModelKeys modelKeys)
+    {
+        return  _properties.get(modelKeys.toString());
+    }
+
+    public IssueModel addAttachment(File attachmentFile)
+    {
+        _attachments.add(attachmentFile);
+        return this;
+    }
+
+    public List<File> getAttachments()
+    {
+        return _attachments;
+    }
+
+    public enum ModelKeys
+    {
+        title(),
+        issueid(),
+        issueDefName(),
+        assignedto(),
+        action(),
+        type(),
+        priority(),
+        comment(),
+        notifyList(),
+        resolution(),
+        resolved(),
+        status();
+
+        ModelKeys()
+        {
+
+        }
```

Comment:
Don't need a constructor for simple enums.
```suggestion
        title,
        issueid,
        issueDefName,
        assignedto,
        action,
        type,
        priority,
        comment,
        notifyList,
        resolution,
        resolved,
        status,
```